describe/Val.ts

```diff
@@ -1,3 +1,11 @@
+/** Sort `ValDescription` by `sGroup` attribute */
+export function sortValues(a: ValDescription, b: ValDescription): number {
```

Comment:
Minor detail, but technically a function with this signature is called a "compare" function, so it would probably be more accurate to call it `compareBySGroup` or `compareValues` in order to more clearly distinguish it from your aptly named `sortENsAttributes` and `sortRecord`, which are technically actual sorting functions.

Reply:
True thanks for the hint